library/ssl_tls13_client.c

```diff
@@ -205,8 +205,8 @@ static int ssl_tls13_write_supported_groups_ext( mbedtls_ssl_context *ssl,
                                                  size_t *olen )
```

Comment:
```suggestion
                                                 size_t *out_len )
```
To expand your work on length variables I'd suggest to change those as well.